pkg/apis/serving/v1alpha1/revision_types.go

```diff
@@ -101,6 +102,15 @@ const (
 	RevisionRequestConcurrencyModelMulti RevisionRequestConcurrencyModelType = "Multi"
 )
 
+// RevisionContainerConcurrencyType is an integer expressing a number of
+// in-flight (concurrent) requests.
+type RevisionContainerConcurrencyType int64
+
+const (
+	// The maximum configurable container concurrency.
+	RevisionContainerConcurrencyMax RevisionContainerConcurrencyType = 1000
```

Comment:
No.  What limit would you recommend?

Reply:
I guess I don't have a good (founded) recommendation. Let's keep it as-is for now, users can always default to unlimited.

Nit: `RevisionContainerConcurrencyType` is int64, Breaker only handles int32, should we adjust or do we not care because we bound to 1000 anyway?

Reply:
I'm presuming that this limit means "you can limit anywhere from 1 - 1000, or you can go bigger by specifying 0". If so, this limit seems like a reasonable sanity limit for now.